client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -358,3 +363,20 @@ export function getArrayOfFilteredTasks( tasks: Task[], flow: string | null ) {
 		}, [] as Task[] )
 	);
 }
+
+// If email is verified, filter out verify_email task
+// If email is not verified, disable all other tasks
+export function updateTasksBasedOnEmailVerification(
```

Comment:
Do we need to expose this method for public use? To me, this could probably be folded into `getArrayOfFilteredTasks` so other consumers don't have to remember to import this method

Reply:
Actually, I initially put this in getArrayOfFilteredTasks(). I pulled it out thinking more about smaller, single-purpose, composable functions. But if we'd rather keep the logic there, I'm ok with that too.